spec/system/page_objects/components/sidebar.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Components
+    class Sidebar < PageObjects::Components::Base
+      def visible?
+        page.has_css?("#d-sidebar")
+      end
+
+      def has_category_section_link?(category)
+        page.has_link?(category.name, class: "sidebar-section-link")
```

Comment:
Neat I didn't realise you could do it this way around, I've been doing has_css? with text: